torch/_torch_docs.py

```diff
@@ -20,6 +20,12 @@ def parse_kwargs(desc):
     return {desc.split(' ')[0]: desc for desc in kwargs}
 
 
+reduceops_common_args = parse_kwargs("""
+    dtype (:class:`torch.dtype`, optional): the desired data type of returned tensor.
```

Comment:
I don't think we should say "the input tensor is casted to dtype before the operation is performed"; that's an implementation detail that we don't want to commit to keeping (it's unnecessarily slow).  The numpy documentation uses better verbiage here: "The type of the returned array and of the accumulator in which the elements are summed." (https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.sum.html)